### PR TITLE
create-wp-project script CLI conversion

### PR DESCRIPTION
**Pull request by IvanGrginovInf**
Integrated https://www.npmjs.com/package/yargs into the script in order to make a proper CLI script out of `create-wp-project`. This was needed for proper integration testing as well as to enable having multiple scripts run from a single package (like setting up a theme or setting up a plugin). 

The following things will be possible after this PR is merged and the NPM package for the install script is updated:

1. You can run `npx create-wp-project --help` to get a list of all available arguments
2. You can provide some or all arguments, you will be prompted for the rest.
3. You can run the script without any prompts at all (by providing the 3 required arguments as well as the summary skip) like so:
    * `npx create-wp-project --projectName="Test Project 1" --url="whatever.local" --description="This is a description" --noSummary`

On top of this, I've added a few more unit tests as well as an integration test that covers the whole install script (it runs `npx create-wp-project` with some default arguments and asserts that the process went smoothly).

After merging and deploying, I'll try making the test run automatically every X hours using github actions.

Changelog
---
* Integrated yargs
* Added an integration tests that tests the whole setup process
* Added a few unit tests
* Simplified the whole setup by removing https://github.com/infinum/eightshift-scripts as a dependancy and adding those things to `create-wp-project` package (this was a legacy thing from when we were planning on having multiple scripts)
* Made changes to the blocks copy process (only some are copied as per https://github.com/infinum/eightshift-frontend-libs/issues/45). I've also added a note at the end of the setup pointing users towards the storybook if they need more blocks.

### CODE REVIEW

File: setup/create-wp-project/src/basics/arguments.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    message: 'Looks good?',
+  });
+
+  return confirmSummary;
+};
+
+/**
+ * Should prompt the user for all scriptArguments.
+ *
+ * TODO: Only prompt for things not provided as arguments on command line.
```
Comment (dingo-d) [inline setup/create-wp-project/src/basics/arguments.js]: If this `TODO` is not implemented, remove it from here and open an issue. 

Inline `TODO`s are never done 😄

File: setup/create-wp-project/src/commands/theme.js [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
+const { copyBlocksFolder } = require('../blocks');
+const { cleanup } = require('../cleanup');
+const { scriptArguments } = require('../arguments');
 
-const { scriptArguments } = require('./src/arguments');
-const { searchReplace } = require('./src/search-replace');
-const { copyBlocksFolder } = require('./src/copy-blocks');
-const { cleanup } = require('./src/cleanup');
+exports.command = '*';
+exports.desc = 'Setup a new WordPress theme (or plugin soon!). Should be run inside your theme folder (wp-content/themes).';
```
Comment (dingo-d) [inline setup/create-wp-project/src/commands/theme.js]:
```suggestion
exports.desc = 'Setup a new WordPress theme (plugin install is coming soon). Should be run inside your theme folder (wp-content/themes).';
```

File: setup/create-wp-project/tests/integration/create-wp-project.test.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  await del(testingFolder);
+}, timeout);
+
+it('tests create-wp-project script (local version) - creates a theme', async() => {
+  const { err } = await exec(`cd ${testingFolder} && node ~/my-packages/eightshift-frontend-libs/setup/create-wp-project/create-wp-project.js --projectName="Test Project" --url="eightshift.local" --description="This is a description" --noSummary`);
+
+  expect(err).toBeFalsy();
+}, timeout);
+
+// it('tests create-wp-project script (npx version) - creates a theme', async() => {
```
Comment (dingo-d) [inline setup/create-wp-project/tests/integration/create-wp-project.test.js]: Commented out code

**Comment by iruzevic** [comment]
Also in the previouse release i saw there are some eslint errors and warnings in the scripts. Could you also fix theme in this branch. Tnx

File: setup/create-wp-project/src/basics/arguments.js (hunk 1 above)
Comment (IvanGrginovInf) [reply setup/create-wp-project/src/basics/arguments.js]: > If this `TODO` is not implemented, remove it from here and open an issue.
> 
> Inline `TODO`s are never done 😄

That's what I just implemented in this branch, forgot to remove the TODO tho, good catch, thx!

**Comment by IvanGrginovInf** [comment]
> Also in the previouse release i saw there are some eslint errors and warnings in the scripts. Could you also fix theme in this branch. Tnx

@iruzevic fixed ALL eslint / stylelint errors, both ones from the script and from the rest of libs (now husky will actually let you commit).

**Pull request merged**